app/controllers/api/v1/slack/user_info_controller.rb

```diff
@@ -0,0 +1,44 @@
+module Api
+  module V1
+    module Slack
+      class UserInfoController < ApplicationController
+        before_action VerifyRequest 
+ 
+        # get all users 
+        def index
+          render json: Users.all
+        end
+
+        # get a single slack user (specifically want :verified and :slack_name)
+        def show
+          user = Users.find_by(email: params[:email])
+          if user 
+            render json: user 
+          else
+            render json: { error: 'No such email record' }, status: :not_found
+          end
+        end
+
+        # update a single slack name by email
+        def update
+          user = User.find(params[:email])
+          if user.update(:slack_name params[:slack_name])
+            render json: user 
+          else
+            render json: { errors: user.errors.full_messages }
+          end
+        end
+      end 
+      
```

Comment:
We would add a `private` here, so that the `verify_request` method is not `public`.  [For example](https://github.com/OperationCode/operationcode_backend/blob/master/app/controllers/api/v1/resources_controller.rb#L51-L67).